$OpenBSD$

add local include and lib dirs
remove additional compiler flags (-flto doesn't work)

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -28,8 +28,11 @@ include_directories(
         src/fheroes2/resource
         src/fheroes2/spell
         src/tools
-        src/fheroes2/system)
+        src/fheroes2/system
+	/usr/local/include)
 
+link_directories(/usr/local/lib)
+
 set(SOURCE_FILES
         src/engine/audio.cpp
         src/engine/audio.h
@@ -394,5 +397,3 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 ")
 
 set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -g -ggdb -fno-omit-frame-pointer -O0 ")
 
-set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3 -flto")
-set(CMAKE_LD_FLAGS_RELEASE "${CMAKE_LD_FLAGS_RELEASE} -mtune=native -flto")
\ No newline at end of file
